tests/test_kafka.py

```diff
@@ -153,30 +149,45 @@ def kafka_broker_dns():
     producer_config = ["--conf", "spark.cores.max=2", "--conf", "spark.executor.cores=2",
                        "--class", "KafkaFeeder"] + common_args
 
-    if KERBERIZED_KAFKA:
+    if kerberized:
         producer_config += kerberos_args
 
-    producer_id = utils.submit_job(app_url=utils._scala_test_jar_url(),
+    producer_id = utils.submit_job(app_url=jar_uri,
                                    app_args=producer_args,
                                    app_name="/spark",
                                    args=producer_config)
 
-    shakedown.wait_for(lambda: producer_launched(), ignore_exceptions=False, timeout_seconds=600)
+    shakedown.wait_for(lambda: _producer_launched(), ignore_exceptions=False, timeout_seconds=600)
     shakedown.wait_for(lambda: utils.is_service_ready(KAFKA_SERVICE_NAME, 1),
                        ignore_exceptions=False, timeout_seconds=600)
 
     consumer_config = ["--conf", "spark.cores.max=4", "--class", "KafkaConsumer"] + common_args
 
-    if KERBERIZED_KAFKA:
+    if kerberized:
         consumer_config += kerberos_args
 
     consumer_args = " ".join([broker_dns, topic, stop_count, kerberos_flag])
 
-    utils.run_tests(app_url=utils._scala_test_jar_url(),
+    utils.run_tests(app_url=jar_uri,
                     app_args=consumer_args,
                     expected_output="Read {} words".format(stop_count),
                     app_name="/spark",
                     args=consumer_config)
 
     utils.kill_driver(producer_id, "/spark")
 
+
+def _producer_launched():
+    return utils.streaming_job_launched(PRODUCER_SERVICE_NAME)
+
+
+def _producer_started():
+    return utils.streaming_job_running(PRODUCER_SERVICE_NAME)
+
+
+def _kafka_broker_dns():
+    cmd = "{package_name} --name={service_name} endpoints broker".format(
+        package_name=KAFKA_PACKAGE_NAME, service_name=KAFKA_SERVICE_NAME)
```

Comment:
The service name may be different in the soak cluster - "kafka-kerberos". Is it configurable in the test?

Reply:
good catch, will change